CREATE_A_ROLE.md

```diff
@@ -1,32 +1,107 @@
 A basic guide for how to create a new role in oasis-roles
 
-Setup environment
-=================
-
-The following Python tools should be installed in either a virtualenv or in your system or elsewise
-should be on your system path:
-
-* ansible
-* ansible-lint
-* yamllint
-* flake8
-* molecule
-
-Create Role
-===========
-
-* `mkdir -p oasis/roles && cd oasis/roles`
-* `git clone git@github.com:oasis-roles/meta_skeleton.git`
-* `ansible-galaxy init --role-skeleton=meta_skeleton my_role_name`
-* `cd my_role_name`
-* Edit README.md, meta/main.yml, and write your code
-
-Setup CI Services
-=================
-
-* Create repo in GitHub
-* Create Travis CI job for new repo
-* Add GH remote and push code
-* `ansible-galaxy import oasis-roles my_role_name` - adds repo to global Galaxy index
-* `ansible-galaxy setup travis oasis-roles my_role_name [Travis CI API key]` - adds auto-updating Galaxy index
-  when Travis CI jobs are completed
+# Setup environment #
+
+* Install [podman](https://podman.io/) using your system package manager.  This
+  example is for Fedora:
+
+  `sudo dnf install podman-docker`
+
+* Create a virtualenv.
+  [virtualenvwrapper](https://virtualenvwrapper.readthedocs.io/en/latest/install.html#basic-installation)
+  is recommended.
+
+  `mkvirtualenv oasis`
+
+* Install Python dependencies.
+
+  `pip3 install ansible tox-ansible pytest-molecule molecule-openstack molecule-podman molecule-docker os-cloud-config`
+
+* Install the OASIS
+  [ansible_collection_system](https://github.com/oasis-roles/ansible_collection_system)
+  collection via Galaxy to use the
+  [molecule_openstack_ci](https://github.com/oasis-roles/ansible_collection_system/tree/master/roles/molecule_openstack_ci)
+  role for OpenStack provisioning.
+
+  `ansible-galaxy collection install git+https://github.com/oasis-roles/ansible_collection_system.git`
```

Comment:
It should be possible to make this just

`ansible-galaxy collection install oasis_roles.system`